rb/lib/selenium/webdriver/bidi/browsing_context.rb

```diff
@@ -94,6 +94,12 @@ def create(type: nil, context_id: nil)
           result = @bidi.send_cmd('browsingContext.create', type: type.to_s, referenceContext: context_id)
           result['context']
         end
+
+        def set_viewport(context_id: nil, width: nil, height: nil, device_pixel_ratio: nil)
```

Comment:
Whats the policy on code documentation - Looking above all other items have it. Not sure if this needs adding here?

Reply:
I think we want to incline more into using RBS as a reference, @p0deje can correct me, but it would be odd for me to have the same twice when RBS shows you in the IDE the expect type

Reply:
We should add params/return and examples in YARD format for public interface, but I would consider everything in `bidi/` an internal implementation detail and not worry about the documentation.